Resources/config/sitemap.xml

```diff
@@ -15,7 +15,7 @@
         <parameter key="cmf_seo.sitemap.guesser.location.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\LocationGuesser</parameter>
         <parameter key="cmf_seo.sitemap.guesser.default_change_frequency.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\DefaultChangeFrequencyGuesser</parameter>
         <parameter key="cmf_seo.sitemap.voter_chain.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\VoterChain</parameter>
-        <parameter key="cmf_seo.sitemap.phpcr.publish_workflow_voter.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\PublishWorkflowVoter</parameter>
```

Comment:
Did we move that voter out of the PHPCR scope? the publish workflow stuff is PHPCR, isn't it?